CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.12)
 
 project(icub_firmware_shared
-        VERSION 1.35.1)
+        VERSION 1.35.2)
```

Comment:
We released v1.36.0 with the latest distro.
Therefore, this needs to be updated to something like 1.36.1, @davidetome.

cc @Nicogene @valegagge

Reply:
I did it via https://github.com/robotology/icub-firmware-shared/pull/87/commits/08e82a88ebc24b60b087feafb9130461842b31b4.